content/advent-2018/go-arrow.md

```diff
@@ -0,0 +1,531 @@
++++
+author = ["Sebastien Binet"]
+title = "Go and Arrow: building blocks for data science"
+linktitle = "go-arrow"
+date = 2018-12-18T00:00:00Z
+series = ["Advent 2018"]
++++
+
+Today we will see how [Arrow](https://arrow.apache.org) could be useful for data science, or -- really -- a lot of analysis' workloads.
```

Comment:
Apache Arrow. You can use "Arrow" after the first mention of "Apache Arrow". see https://www.apache.org/foundation/marks/#books

Reply:
"analysis" -- lose the apostrophe